packages/ui/src/ui/pages/query-tracker/QueryResultsView/YQLTable/YQLTable.tsx

```diff
@@ -73,6 +76,12 @@ type Props = {
     onSort?: (sortOrder?: SortOrder | SortOrder[]) => void;
     showFullValueInDialog?: boolean;
     containerRef?: React.Ref<HTMLDivElement>;
+    queryParams: TableQueryParams;
```

Comment:
Do we really need to know about `queryParams` on this level? Isn't it better to pass callback `onShowPreveview`?
```ts
onShowPreview: (colName: string, rowIndex: number) => void;
```